cla_backend/apps/reports/views.py

```diff
@@ -193,31 +201,11 @@ def download_file(request, file_name="", *args, **kwargs):
     response["Content-Disposition"] = "attachment; filename=%s" % smart_str(file_name)
     response["X-Sendfile"] = smart_str("%s%s" % (settings.TEMP_DIR, file_name))
 
-    try:
-        export_record = Export.objects.get(user_id=request.user.pk, path__endswith=file_name)
-        export_record.delete()
-    except Export.DoesNotExist:
-        raise Http404("Export does not exist")
+    if not file_name.__contains__("scheduled"):
```

Comment:
I was half way through writing "it's not clear to me why we only have to get the file if it's a scheduled one" when I realised what was actually going on.
Granted, I wasn't reading the whole method in order or I might have twigged, but maybe moving the try/except into something like a `self.delete_export()` method might make it super clear what's happening